services/cd-service/pkg/repository/transformer.go

```diff
@@ -609,13 +627,13 @@ func (c *CreateApplicationVersion) Transform(
 	sortedKeys := sorting.SortKeys(c.Manifests)
 
 	if state.DBHandler.ShouldUseOtherTables() {
-		anyRelease, err := state.DBHandler.DBSelectAnyRelease(ctx, transaction)
+		anyRelease, err := state.DBHandler.DBSelectReleasesByApp(ctx, transaction, c.Application, false)
 		if err != nil {
```

Comment:
DBSelectAnyRelease Did not make any sense here? Any release simply checks for existence of any release for any app on any env, and we then make decisions based on the result, which cause problems with previously deleted applications

Reply:
And I don't know how it didn't come up earlier